### DISCUSSION TITLE
Support torch tensors

### DISCUSSION THREAD

**Discussion by ptoupas** [francof2a/fxpmath#83] [General]
Hi,

First of all i would like to comment that the repo is very useful and super easy to use as well as very well written! Amazing work!

I was wondering if you are planning on supporting torch tensors as input to Fxp alongside with the rest of the supported formats and types.

Thanks

**Comment by francof2a** [comment]
Hello,

Thank you very much for your comments! I'm glad to hear about `fxpmath` is useful.

About `torch` tensor, the general answer is yes, but "support" is a quite broad concept. Because I have few time, maybe the new features are not coming soon.

In the meantime, it would be nice to know if you have in mind some features that you consider most important to supporting `torch`.

Let me move this issue into **Discussions** panel.
